### DISCUSSION TITLE
Different behavior of performance in local and stage

### DISCUSSION THREAD

**Discussion by emadimehrdad** [getsentry/sentry-python#1942] [Q&A]
Hello
I want to use performance in sentry. I instrumented it in my code (python/Django) and it worked on my local system. But it does not work on stage and it does not get any transactions. It gets issues and it works properly. I just have this problem with my performance tab. I searched a lot but I could not find any solution to solve this problem.

Here is my settings.py code:

```
    sentry_sdk.init(
        dsn=SENTRY_DSN,
        environment=SENTRY_ENVIRONMENT,
        attach_stacktrace=True,
        with_locals=True,
        integrations=[DjangoIntegration(), CeleryIntegration()],
        server_name='',
        send_default_pii=True,
        before_send=sentry_exception_handler,
        traces_sample_rate=1.0,
    )
```

**Comment by smeubank** [comment]
hi @emadimehrdad 

could you try turning the [debug option](https://docs.sentry.io/platforms/python/guides/logging/configuration/options/#debug) on in prod temporarily? We might be able to gather some insights there from SDK debug logs.


Do you have any upstream services like a JS frontend which you are connecting with? 

edit: checking for [folks using logging/debug](https://stackoverflow.com/questions/66888438/disable-sentry-logs). this SO question might be helpful. You may need to adjust [logging config](https://develop.sentry.dev/config/#logging) as well, in case you don't see information you need right away

**Reply by emadimehrdad** [reply]
Thanks for your reply. I did your suggestions and answered them in another discussion.
And yeah. We have JS frontend, but I think it does not matter. Because I test my APIs in graphiql which is a backend component.
